submodules/privatelink/outputs.tf

```diff
@@ -0,0 +1,6 @@
+output "info" {
+  description = "Target groups..."
```

Comment:
if you can, please fill this in before merging

Reply:
it only return target groups, I added "..." based on other outputs that use the same pattern, I can remove it if you refer to that